server/associations.js

```diff
@@ -1,26 +1,13 @@
-const Groups = require('./models/index.js').Groups;
-const Members = require('./models/index.js').Members;
-const Messages = require('./models/index.js').Messages;
-const GroupMembers = require('./models/index.js').GroupMembers;
-const UserGroup = require('./models/index.js').UserGroup;
-const Viewers = require('./models/index.js').Viewers;
-const GroupMessages = require('./models/index.js').GroupMessages;
+import db from './models/index';
 
-module.exports = {
-  associate: () => {
-    Members.belongsToMany(Groups, { through: GroupMembers, as: 'joinedGroups' });
-    Members.belongsToMany(Groups, { through: UserGroup, as: 'createdGroups' });
-    Members.sync();
-
-    Groups.belongsToMany(Members, { through: GroupMembers, as: 'groupMembers' });
-    Groups.belongsToMany(Members, { through: UserGroup, as: 'admin' });
```

Comment:
Are you working on a Post App? Why do you have to remove these blocks?

Reply:
Was working on it before coming to Bootcamp. PostIt